packages/manager/modules/iam/src/components/resourceSelect/resourceSelect.controller.js

```diff
@@ -98,7 +101,14 @@ export default class ResourceSelectController {
     return this.form?.[this.resourceTypesName];
   }
 
-  $onChanges({ required }) {
+  $onChanges({ ngModel, required }) {
+    if (ngModel) {
+      this.model = cloneDeep(ngModel.currentValue);
```

Comment:
duable ?

```suggestion
      this.model = {...ngModel.currentValue};
```

Reply:
Nope, the model contains objects that need to be cloned as well